hummingbot/connector/exchange/bitfinex/bitfinex_api_user_stream_data_source.py

```diff
@@ -49,12 +47,8 @@ async def listen_for_user_stream(self, ev_loop: asyncio.BaseEventLoop, output: a
                 await ws.authenticate()
 
                 async for msg in ws.messages():
-                    transformed_msg: BitfinexOrderBookMessage = self._transform_message_from_exchange(msg)
-
-                    if transformed_msg is None:
-                        continue
-                    else:
-                        output.put_nowait(transformed_msg)
+                    if msg[1] not in [ContentEventType.HEART_BEAT, ContentEventType.AUTH, ContentEventType.INFO]:
```

Comment:
Doesnt seem like we are using `HEART_BEAT` anywhere. Let's remove it if we're not using it.
Looking at the docs, they did not specify any keep-alive mechanism anyway.

Reply:
I think the heartbeat message is excluded in the function `_prepare_trade` in `BitfinexAPIOrderBookDataSource`. And it seems like we can't chose to not receive that message in the websocket connection (https://docs.bitfinex.com/docs/ws-general#heartbeating)

Reply:
Yea I dont mind that we are receiving that heartbeat message, I would rather we do not queue the `HEART_BEAT` messages if we're not using it.